### ISSUE TITLE
Regularization Section: Incorrect code snippet

### ISSUE THREAD

**Issue by qherm** [natolambert/rlhf-book#115]
In Step 2 of the first code snippet, you use `inputs` instead of `generated_tokens` to get the logits. You're looking to get the probability of the sequence, not of the inputs; so it should use `generated_tokens`

**Comment by natolambert** [comment]
oh thanks @qherm ! Yes
